ui/oopsyraidsy/data/06-ew/trial/asura.ts

```diff
@@ -6,7 +6,35 @@ export type Data = OopsyData;
 
 const triggerSet: OopsyTriggerSet<Data> = {
   zoneId: ZoneId.TheGildedAraya,
-  damageWarn: {},
+  damageWarn: {
+    'Asura Chakra Ring 1': '8C9B', // ring AOE
+    'Asura Chakra Ring 2': '8C9C', // ring AOE
+    'Asura Chakra Ring 3': '8C9D', // ring AOE
+    'Asura Chakra Ring 4': '8C9E', // ring AOE
+    'Asura Chakra Ring 5': '8C9F', // ring AOE
+    'Asura Laceration': '8C97', // clone puddle AOEs
+    'Asura Pedestal Purge': '8C81', // circle AOE under boss
+    'Asura Wheel of Deincarnation': '8C83', // donut AOE under boss
+    'Asura Bladewise': '8C85', // line cleave from boss
+    'Asura Image - Pedestal Purge': '8C82', // big circle AOE under image
+    'Asura Image - Wheel of Deincarnation': '8C84', // big donut AOE under image
+    'Asura Image - Bladewise': '8C86', // big line cleave from image
+    'Asura Khadga 1': '8C89', // half-room cleave
+    'Asura Khadga 2': '8C8A', // half-room cleave
+    'Asura Khadga 3': '8C8B', // half-room cleave
+    'Asura Khadga 4': '8C8C', // half-room cleave
+    'Asura Khadga 5': '8C8D', // half-room cleave
+    'Asura Khadga 6': '8C8E', // half-room cleave
+    'Asura Face of Wrath': '8CA6', // red/blue mechanic (red unsafe)
+    'Asura Face of Delight': '8CA7', // red/blue mechanic (blue unsafe)
+    'Asura Myriad Aspects 1': '8CB4', // fan dodge (first hit)
+    'Asura Myriad Aspects 2': '8CB5', // fan dodge (second hit)
+    'Asura Scattering': '8C99', // rotating line cleaves
+  },
+  shareFail: {
+    'Asura Cutting Jewel': '8CA0', // tankbuster
+    'Asura Ordered Chaos': '8CA3', // spread AOEs during Chakra
```

Comment:
Oh, I missed this earlier, but this should probably be a `shareWarn`, not a failure. It doesn't inflict a vulnerability like it would on EX or Savage, and it doesn't really do all that much damage compared to harder content.